wdio.conf.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const ExpressDevService = require('terra-toolkit/lib/wdio/services/index').ExpressDevService;
+const localIP = require('ip');
+const webpackConfig = require('./src/config/webpack.config');
+
+const config = {
+  ...wdioConf.config,
+
+  baseUrl: `http://${localIP.address()}:${8080}`,
+
+  // Configuration for ExpressDevService
+  webpackConfig,
+
+  // Travis only has 1 browser instace, set maxInstances to 1 to prevent timeouts
+  maxInstances: process.env.CI ? 1 : wdioConf.config.maxInstances,
```

Comment:
we should remove `maxInstances`. In [toolkit v2.9.1](https://github.com/cerner/terra-toolkit/pull/71) we updated this default value due to some selenium:hub issues we were seeing locally for large test sets. Not an issue here at this point, but it will be good to be consistent.